exporter/clickhouselogsexporter/exporter.go

```diff
@@ -445,11 +466,18 @@ func (e *clickhouseLogsExporter) pushToClickhouse(ctx context.Context, ld plog.L
 
 		for bucketTs, resources := range resourcesSeen {
 			for resourceLabels, fingerprint := range resources {
+				// if a resource fingerprint is seen in the bucket already, skip inserting it again.
+				key := makeKeyForRFCache(bucketTs, fingerprint)
+				if e.rfCache.Get(key) != nil {
```

Comment:
Q: Will this update the ttl for the cache key when there is a hit ? if yes then it might be a problem as it won't be removed ever if the key is continiously

Reply:
See the init part
```
ttlcache.WithDisableTouchOnHit[string, struct{}](),
```